vertical-pod-autoscaler/go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/kylelemons/godebug v1.1.0 // indirect
```

Comment:
Any idea why this is being added?

Reply:
Seems to be added because of the `github.com/prometheus/client_golang` import in the new tests: https://github.com/prometheus/client_golang/blob/a4af5c85ff987d05e390802916f4d8b5bc7b61a2/go.mod#L11